spec/webauthn/attestation_statement/tpm_spec.rb

```diff
@@ -250,6 +254,8 @@
         end
 
         it "works if everything's fine" do
+          pending("Waiting for PSS support in the tpm-key_attestation gem")
```

Comment:
Remove this once https://github.com/cedarcode/tpm-key_attestation/issues/1 is done.